ScanbotSDKexample/camera-fragment/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="io.scanbot.example">
+
+    <application
+        android:allowBackup="true"
+        android:name=".ExampleApplication"
+        android:label="@string/app_name"
+        android:theme="@style/AppTheme"
+        tools:replace="android:theme">
+
+        <!-- TODO
+        <meta-data android:name="SCANBOT_SDK_LICENSE_KEY" android:value="Insert your key here" />
+        -->
+
+        <activity
+            android:name=".MainActivity"
+            android:label="@string/app_name">
```

Comment:
Explicitly add the `ScanbotTheme` here. It's not obvious that it's needed and even less obvious that `AppTheme` extends from it.